### PR TITLE
removeadd function 80% complete

### PR DESCRIPTION
**Pull request by 21ftt1750**
focus other function first

add checlist item and remove item checklist

### CODE REVIEW

**Review by anwari-fikri — CHANGES_REQUESTED** [review]
- [x] resolve all problems i.e. Typescript type error

File: app/homepage/page.tsx [hunk 1]
```diff
@@ -1,116 +1,117 @@
...
-  
-  const [isDialogOpen, setIsDialogOpen] = useState(false);
-
-  const handleSubmit = (event: { preventDefault: () => void; }) => {
-    event.preventDefault();
-
-    const newProject = {
-        id: projectName,
+  const LandingPage = () => {
+    const [projectName, setProjectName] = useState('');
```
Comment (anwari-fikri) [inline app/homepage/page.tsx]: add object types to variables/constant

eg. [projectName, setProjectName] = useState\<string\>("");

^Less headache in the future

File: app/homepage/page.tsx [hunk 2]
```diff
@@ -1,116 +1,117 @@
...
 
-  const handleRemoveProject = (projectId) => {
-    setProjects((prevProjects) => prevProjects.filter((project) => project.id !== projectId));
-  };
+    return (
+      <div className='bg-[#00040D] w-screen h-screen relative flex justify-center items-start pt-48 text-white'>
+        <Link href='/login' className='absolute top-0 left-0 mt-12 ml-16 flex items-center'>
+          <ChevronLeft className='size-8' /> <Image src={logo} alt='' className='size-16' />
+        </Link>
+        <div className='absolute top-0 left-0 w-full flex justify-center pt-14'>
```
Comment (anwari-fikri) [inline app/homepage/page.tsx]: try to not use absolute.

use flexbox instead

File: app/project/page.tsx [hunk 3]
```diff
@@ -34,22 +56,62 @@ const Page = () => {
       </div>
       <div className='relative'>
         <Image src={background} alt={'background'} />
```
Comment (anwari-fikri) [inline app/project/page.tsx:58]: using image as background might cause layout issue in the future

File: app/homepage/page.tsx (hunk 1 above)
Comment (21ftt1750) [reply app/homepage/page.tsx]: [51832115154a23aca88249d3dad51b48d767ec99](https://github.com/21ftt1750/tododo/pull/2/commits/51832115154a23aca88249d3dad51b48d767ec99)

fixed

File: app/homepage/page.tsx (hunk 2 above)
Comment (21ftt1750) [reply app/homepage/page.tsx]: https://github.com/21ftt1750/tododo/pull/2/commits/9fdd58390e30c01905e008d61ccde062f78f372b

fixed

File: app/project/page.tsx (hunk 3 above)
Comment (21ftt1750) [reply app/project/page.tsx:58]: the whole picture is the background, including the pencil, paper and wordings. How may i change this
project

Comment (anwari-fikri) [reply app/project/page.tsx:58]: oops this is just meant as a heads up. can merge now.

**Review by anwari-fikri — APPROVED** [review]

**Pull request merged**